test/core/ticktree/TickTreeUpdateOffer.t.sol

```diff
@@ -185,16 +185,18 @@ contract TickTreeUpdateOfferTest is TickTreeTest {
     uint[] storage higherBinListSizeScenarios,
     uint[] storage lowerBinListSizeScenarios
   ) internal {
-    vm.pauseGasMetering();
+    // vm.pauseGasMetering();
     runBinScenarios(bin, higherBinListSizeScenarios, lowerBinListSizeScenarios);
-    vm.resumeGasMetering();
+    // vm.resumeGasMetering();
```

Comment:
These commented out lines should be deleted, no?

Reply:
yes! fixed in https://github.com/mangrovedao/mangrove-core/pull/579